admin_commands/tremove.js

```diff
@@ -0,0 +1,28 @@
+// tremove.js
+// Clears topic at input index.
+module.exports.description = "Clears topic at the index given."
+
+const topic0p = require("../lib/topic.js");
+
+module.exports.run = (client, message, args) => {
+	const channel = message.channel;
+
+	// Get the index of topic we want removed
+	indexToClear = parseInt(args);
```

Comment:
Since `removeReactionToInactiveTopic` supports removing *all* inactive topics, it would be a good idea to support removing multiple topics at once.

Reply:
indeed - please tell our scrum master to create a new story for that & point it up